addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -589,8 +675,16 @@ const Wysiwyg = Widget.extend({
  * @returns {Node} ec - end container
  * @returns {Number} eo - end offset
  */
-Wysiwyg.getRange = function (node) {
+Wysiwyg.getRange = function (node = undefined) {
```

Comment:
agreed, 

I was afraid it would break some Qunit tests if we remove the argument altogether. So i just made it optionnal
It was a bit lazy, I know.